src/controllers/destination.ts

```diff
@@ -138,6 +140,12 @@ export default class DestinationController {
       'Native(Router-Transform):: Response from transformer::',
       JSON.stringify(ctx.body),
     );
+    stats.timing('dest_transform_request_latency', startTime, {
+      destination,
+      version: 'v0',
+      type: 'router',
```

Comment:
Instead of `type` let's put it as `feature`(common convention)

Reply:
OKay

Reply:
Done